Tabloid/client/src/modules/categoryManager.js

```diff
@@ -19,6 +20,23 @@ export const getAllCategories = () => {
   });
 };
 
+export const getCategoryById = (id) => {
+  return getToken().then((token) => {
+    return fetch(`${_apiUrl}/${id}`, {
+      method: "GET",
+      headers: {
+        Authorization: `Bearer ${token}`,
+      },
+    }).then((res) => {
+      if (res.ok) {
+        return res.json();
+      } else {
+        throw new Error("Unknown error getting categories");
```

Comment:
was the error getting categories? or a single category?

Reply:
Fixed